packages/SwingSet/test/basedir-promises-3/vat-right.js

```diff
@@ -9,7 +9,7 @@ export function buildRootObject() {
       // arg should be a Promise that promptly resolves to 4
       const argP = Promise.resolve(arg);
       const wasP = argP === arg;
-      argP.then(newArg => pk4.resolve([wasP, newArg]));
+      return argP.then(newArg => pk4.resolve([wasP, newArg]));
```

Comment:
ok, that's ok for similar reasons: bootstrap.js is not explicitly doing anything with the promise it gets back from `E(p2).four()`, so having the resolution timing change doesn't affect the thing being tested. Some of these tests are pretty subtle, which is why I'm looking at them so carefully.